### DISCUSSION TITLE
Change presence of Discord Bot

### DISCUSSION THREAD

**Discussion by demetera** [krisppurg/dimscord#98] [Plans]
Hi,

is there  any way to change status of the bot itself (e.g. set it to Invisible)?
in Python's library there's :  async def change_presence() available

Thanks!

**Comment by krisppurg** [comment]
[yes](https://krisppurg.github.io/dimscord/dimscord/gateway.html#updateStatus%2CShard%2Cseq%5BActivityStatus%5D%2Cstring)

**Reply by demetera** [reply]
Thanks a lot. Didn't search through the entire docs carefully
